integration/crd.go

```diff
@@ -38,11 +38,10 @@ metadata:
   name: test-attestor
 spec:
   noteReference: projects/%s
-  privateKeySecretName: %s
   publicKeyData: %s`
 )
 
-// Secret name for test-attestor
+// Secret name for test-attestor, which matches integration/testdata/image-security-policy/my-isp.yaml
```

Comment:
Please consider moving the key pair generation out of createAA, because AA now only has the public portion.  One way to do it:
- move createAA call out of here and into setupISP() in integration/run_test.go
- rename this to createSecret or something like it
  + and change this to return the generated public key

WDYT?

Reply:
Done.